packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -19,7 +19,7 @@
         }
         &-scrollbar {
             position: absolute;
-            height: calc(100% - 2.4rem - 72px - 92px);
+            height: calc(100% - 2.4rem - 92px);
```

Comment:
wuld be good if we have sass named variable for these, 2.4rem and 92px

Reply:
We're not re-using this anywhere else so I don't see the need for keeping a variable at this time.